examples-standalone/dashboard/src/main.js

```diff
@@ -0,0 +1,12 @@
+import { createApp } from 'vue'
+// Import our custom CSS
+import './scss/styles.scss'
+
+// Import all of Bootstrap's JS
+import * as bootstrap from 'bootstrap'
```

Comment:
Do we need the Bootstrap's JS for anything?

Reply:
fixed